lib/helpers.js

```diff
@@ -2,10 +2,8 @@
 
 var assert = require('assert');
 
-var filter = require('arr-filter');
-var map = require('arr-map');
+// bach supports nodejs>=v10.13, but Array#flat is introduced in v11.
 var flatten = require('arr-flatten');
```

Comment:
Should `arr-flatten` be replaced by `Array.prototype.slice.apply`?

Reply:
`arr-flatten` expands array elements recursively.